tests/pyreverse/test_main.py

```diff
@@ -0,0 +1,25 @@
+import os
+import sys
+
+import pytest
+
+from pylint.lint import fix_import_path
+
+TEST_DATA_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "data"))
+PROJECT_ROOT_DIR = os.path.abspath(os.path.join(TEST_DATA_DIR, ".."))
+
+
+@pytest.fixture(name="setup_path", params=[PROJECT_ROOT_DIR, TEST_DATA_DIR])
```

Comment:
Why not name the fixture ``setup_path`` directly ?

Reply:
That was the plan initially but I see pylint gives: 'W0621: Redefining name 'setup_path' from outer scope (line 13) (redefined-outer-name)'.

From the the pytest docs:
```
If a fixture is used in the same module in which it is defined, the function name of the fixture will be shadowed by the function arg that requests the fixture; one way to resolve this is to name the decorated function fixture_<fixturename> and then use @pytest.fixture(name='<fixturename>').
```
https://docs.pytest.org/en/stable/reference.html#pytest-fixture

Reply:
You could move the fixture definition into ``conftest.py`` to resolve this. 
An alternative (@Pierre-Sassoulas) would be to use ``pylint-pytest``, which suppresses among others this.

Reply:
Let's do it ! On top of removing false positive, we'll eat our own dog food and this will be a compatibility tests for the latest pylint too.

Reply:
Thanks for this idea @DudeNr33 . I'm happy to use this approach.
I now realise that the test can have the same result but simplified, here is the update while using the earlier approach from @Pierre-Sassoulas : 37f3a5005771bffa8a6539b4f4b70f56fe933d0b